spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -174,6 +174,29 @@ be `TaskLauncher` implementation specific.
 dataflow:>task launch mytask --properties "deployer.timestamp.foo1=bar1,app.timestamp.foo2=bar2"
 ```
 
+==== Common application properties
+
+In addition to configuration via DSL, Spring Cloud Data Flow provides a mechanism for setting common properties to all
+the task applications that are launched by it.
+This can be done by adding properties prefixed with `spring.cloud.dataflow.applicationProperties.task` when starting the server.
+When doing so, the server will pass all the properties, without the prefix, to the instances it launches.
+
+For example, all the launched applications can be configured to use the properties `foo` and `fizz` by launching the Data Flow server
+with the following options:
+
+```
+--spring.cloud.dataflow.applicationProperties.task.foo=bar
+--spring.cloud.dataflow.applicationProperties.task.fizz=bar2
+```
+
+This will cause the properties `foo=bar` and `fizz=bar2` to be passed to all the launched applications.
+
+[NOTE]
+Properties configured using this mechanism have lower precedence than task deployment properties.
```

Comment:
Hmmm, not sure. While I see what you mean, it is indeed deployment properties, as in [here](https://github.com/spring-cloud/spring-cloud-deployer/blob/master/spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/core/AppDeploymentRequest.java#L62)